atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/TransactionManager.java

```diff
@@ -23,8 +23,10 @@
 import com.palantir.lock.LockRequest;
 import com.palantir.lock.LockService;
 import com.palantir.lock.v2.TimelockService;
+import com.palantir.processors.AutoDelegate;
 import com.palantir.timestamp.TimestampService;
 
+@AutoDelegate(typeToExtend = TransactionManager.class)
```

Comment:
Sounds good, I like this solution much more than having to put it on just one extending class, although it does read strangely since I'm used to the other way.